src/main/java/com/mixpanel/android/mpmetrics/MixpanelFCMMessagingService.java

```diff
@@ -260,6 +279,17 @@ protected static Notification makeNotificationSDK26OrHigher(Context context, Pen
                 setStyle(new Notification.BigTextStyle().bigText(notificationData.message)).
                 setChannelId(channelId);
 
+        for (int i = 0; i < notificationData.buttons.size(); i++) {
```

Comment:
why not adding this feature to API 20 and above instead of 26 and above?

Reply:
idk i was waiting for you to tell me what to do, so should I refactor it to a utility function and call it in both appropriate `makeNotification` functions?